content/user/ui/report/Vulnerability-trends.md

```diff
@@ -0,0 +1,71 @@
+
+<a href="assets/images/vulnerability-trend-report-name.png" rel="lightbox" title="Save a report"><img class="thumbnail" src="assets/images/vulnerability-trend-report-name.png"/></a> 
+ 
+To save a report view, click the star icon at the top right of the graphs. This generates a popup with a field to name the report. Once saved, the named report appears next to the **Vulnerability Trend** heading with a dropdown menu. Each time you come to the Vulnerability Trend page, the menu shows all of your saved reports as well as an option to **Start a new report**. 
+
+<a href="assets/images/Vulnerability-trend-save-report.png" rel="lightbox" title="Menu of saved reports"><img class="thumbnail" src="assets/images/Vulnerability-trend-save-report.png"/></a> 
+
+### Rename reports
+
+When viewing a saved report, hover over the star icon to generate a **Manage Report** tooltip. Click the icon to produce a popup with a field to rename the report and buttons to **Cancel**, **Remove** or **Save**. 
+
+### Edit and remove reports
+
+If you change filter options while viewing a saved report, the star icon appears and the report name is updated as **Edited**. Click the icon to generate a popup menu to **Save Existing** or **Save As New**. Choose **Save Existing** to update the saved report name with the current filters and remove the **Edited** status. Choose **Save As New** to save the report view with the current filters as a new report under a different name.
+
+Click **Remove** to permanently delete the saved report that you're currently viewing. Contrast automatically takes you to the default Vulnerability Trend page view and removes the report name from the dropdown menu. 
+
+### Start new reports
+
+To clear unsaved edits to an existing report and start over with the report defaults, choose the **Start a new report** option in the dropdown menu. The report name changes to **New Report**.
+
+### Manage reports 
+
+In the dropdown menu, click the **Manage** link to rename or remove saved reports. Select the checkbox by each report that you want to rename or remove in the **Manage Saved Reports** dialog. You can also select a checkbox to select all saved reports and edit as a batch action. 
+
+## Export Reports
+
+Create a timestamped PDF report of the Vulnerability Trend to capture a snapshot your vulnerability management. When you click the **Export** icon in the upper right hand corner of the page, Contrast immediately starts generating the report and prompts you to download when it’s ready.
+ 
+Each PDF report provides the chart that you see in the User interface along with a chart that summarizes the variables included in your customized view. The report also summarizes the metrics and breakdowns of each data point. 
```

Comment:
"Each PDF report includes a summary of the variables included in your customized view, as well as the trend graphic and a table of the metrics and breakdowns of each data point."